docs/index.md

```diff
@@ -0,0 +1,83 @@
+# statistic-test
+
+[![Freqtrade CI](https://github.com/PySATL/pysatl-experiment/workflows/PySATL%20CI/badge.svg)](https://github.com/PySATL/pysatl-experiment/actions)
+[![Coverage Status](https://coveralls.io/repos/github/PySATL/pysatl-experiment/badge.svg?branch=main)](https://coveralls.io/github/PySATL/pysatl-experiment?branch=main)
+[![Documentation](https://readthedocs.org/projects/pysatl/badge/)](https://pysatl.github.io/pysatl-experiment/)
+
+This is a test framework for goodness-of-fit statistic tests.
+
+## Architecture
+
+Framework consists of 5 modules
+
+1. Core module - provides distributions, cdf, pdf etc.
+2. Persistence module - provides different stores to store data.
+3. Experiment module - provides pipeline for experiment and default components for pipeline.
+4. Expert system module - provides expert system for goodness-of-fit testing.
+5. Tests module - provides different goodness-of-fit tests.
+
+### Experiment architecture
+
+![PYSATL architecture](pysatl_flow.png "PYSATL architecture")
```

Comment:
Seems like the [pysatl_flow.png](https://github.com/PySATL/pysatl-experiment/blob/main/pysatl_flow.png) file must be added to ```docs``` directory, or a link to the file must be specified to display the image.